visionmetrics/caption/azure_openai_model_eval_base.py

```diff
@@ -0,0 +1,161 @@
+import torch
+from collections import Counter
+from enum import Enum
+from irisml.tasks.create_azure_openai_chat_model import OpenAITextChatModel
```

Comment:
There is also `irisml.tasks.create_azure_openai_completions_model` but looking at the code it look like both call the /completions API?

Reply:
The completions and chat functionalities are different, and our methods use the chat function (the completions function is more outdated and not instruction-tuned, and the newer models like GPT-4o don't support it). The chat completions API is /chat/completions, while the /completions API is different (https://github.com/microsoft/irisml-tasks-azure-openai/blob/main/irisml/tasks/create_azure_openai_completion_model.py#L157 vs https://github.com/microsoft/irisml-tasks-azure-openai/blob/main/irisml/tasks/create_azure_openai_chat_model.py#L81).

Reply:
Aha! Those two are indeed different :).